lib/board.rb

```diff
@@ -0,0 +1,49 @@
+# rubocop: disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Style/MultipleComparison
+class Board
+  attr_accessor :player, :sign, :position
+
+  def initialize
+    @board_data = ['.', '.', '.', '.', '.', '.', '.', '.', '.']
+    @taken = []
+    @total_plays = 0
+    @status = 0
+  end
+
+  def display(board_data)
+    y = 0
+    while y < board_data.length
+      if y == 2 || y == 5
+        puts "  #{board_data[y]}  "
+        puts '----------------'
+      elsif y == 4 || y == 1 || y == 7
+        print " | #{board_data[y]} | "
+      else
+        print "  #{board_data[y]}  "
+      end
+      y += 1
+    end
+    puts "\n \n"
+  end
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- Refactor the display method so instead of being in charge of printing the board is in charge only to retrieve the board in a string that can be printed in the `main.rb` file doing something like: `puts play_ground.display(board_data)`